config/jobs/kubernetes-sigs/cluster-api/cluster-api-upgrades-main.yaml

```
@@ -0,0 +1,174 @@
+periodics:
+- name: periodic-cluster-api-e2e-workload-upgrade-1-17-1-18-main
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  optional: true
+  always_run: false
+  branches:
+  # The script this job runs is not in all branches.
+  - ^master$
+  path_alias: sigs.k8s.io/cluster-api
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-cc9840e-1.19
+      args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
+          value: "v1.17.2"
```

```suggestion
          value: "v1.17.17"
```
Nb. we should make sure that there is an existing kinder node image for each version